C/CUTEst/build_tarballs.jl

```diff
@@ -104,14 +119,21 @@ platforms = expand_gfortran_versions(filter!(!Sys.iswindows, supported_platforms
 
 # The products that we will ensure are always built
 products = [
-    ExecutableProduct("sifdecoder", :sifdecoder),
+    FileProduct("lib/libsifdecode.a", :libsifdecode_a),
+    FileProduct("lib/libcutest_single.a", :libcutest_single_a),
+    FileProduct("lib/libcutest_double.a", :libcutest_double_a),
```

Comment:
I don't think we need to export those.

Reply:
I would like to compile GALAHAD for @tmigot and it could be easier to link with static libraries before we use the "trick" to generate dynamic libraries.